### ISSUE TITLE
Pretrained model of BebyGAN for upsacling factor 2 and 3

### ISSUE THREAD

**Issue by TriB90** [dvlab-research/Simple-SR#16]
Please provide us the pretrained model of BebyGAN for upsacling factors 2 and 3.

**Comment by fenglinglwb** [comment]
Sorry. Following previous work, we only conducted experiments on the x4 setting.

**Comment by TriB90** [comment]
@fenglinglwb Is it possible to train the model for upscaling factors 2 and 3 on our own?

**Comment by fenglinglwb** [comment]
Sure. You could first train an RRDB backbone using L1 loss for initialization, and then train the initialized model using GAN loss.

**Comment by TriB90** [comment]
@fenglinglwb Thank you so much for your prompt reply. What changes do we need to make to the training code if we want to train the model on our own dataset?
